north/src/runtime/mod.rs

```diff
@@ -113,6 +113,13 @@ pub async fn run(config: &Config) -> Result<()> {
         state.applications.len()
     );
 
+    // Handle SIGINT and SIGTERM
+    let tx_interrupt = tx.clone();
+    ctrlc::set_handler(move || {
```

Comment:
This won't compile if feature `termination` is not enabled and it's not enabled by default.

Reply:
It does compile without the `termination` feature. It will just not handle `SIGTERM` as well.